pkg/kubelet/kubelet_node_status.go

```diff
@@ -115,7 +115,7 @@ func (kl *Kubelet) tryRegisterWithApiServer(node *v1.Node) bool {
 		return false
 	}
 
-	clonedNode, err := conversion.NewCloner().DeepCopy(existingNode)
+	clonedNode, err := api.Scheme.DeepCopy(existingNode)
```

Comment:
@Random-Liu @mikedanese

Reply:
Why not just use the generated deep copy directly, e.g. `existingNode.DeepCopy()`, and avoid having to handle an error? See here: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/zz_generated.deepcopy.go#L2941

Reply:
updated, thanks, ptal